lib/std/fs/File.zig

```diff
@@ -1480,7 +1481,7 @@ pub const Reader = struct {
         return n;
     }
 
-    pub fn read(r: *Reader, dest: []u8) std.Io.Reader.Error!usize {
+    fn read(r: *Reader, dest: []u8) std.Io.Reader.Error!usize {
```

Comment:
Is this function actually used anywhere anymore or can it be deleted? From a quick glance I only see direct calls to `readPositional()` and `readStreaming()` in this file.